rslearn/data_sources/gcp_public_data.py

```diff
@@ -347,6 +367,19 @@ def get_item_by_name(self, name: str) -> Sentinel2Item:
             raise ValueError(f"IMAGE_FILE is empty for {name}")
         blob_prefix = base_url + elements[0].text.split("B01")[0]
 
+        # Some Sentinel-2 scenes in the bucket are missing a subset of image files. So
```

Comment:
Suggest factoring this into a clearly named helper function to improve readability